docs/security/host-certs.md

```diff
@@ -27,25 +33,28 @@ The OSG PKI Command Line Clients are tested to work on Python version 2.4+. They
 -   the name of the **Virtual Organization** affiliated with the Certificate Authority
 
 Requesting host/service certificate using OIM
-==================================================
+----------------------------------------------
 
-The OSG PKI Certificate Request & Management System can be found at: <https://oim.opensciencegrid.org/oim/certificate>.
+The OSG PKI Certificate Request & Management System can be found at:
+<https://oim.opensciencegrid.org/oim/certificate>.
 
-For instructions on how to request a host or service certificate using the Web interface please see the [user guide maintained by GOC](https://confluence.grid.iu.edu/pages/viewpage.action?pageId=3244064).
+For instructions on how to request a host or service certificate using the Web
+interface please see the [user guide maintained by
+GOC](https://confluence.grid.iu.edu/pages/viewpage.action?pageId=3244064).
```

Comment:
This is maintained by the "OIM development team", not GOC.

Reply:
Changed